src/Symfony/Bridge/Twig/Command/DebugCommand.php

```diff
@@ -50,19 +52,24 @@ protected function configure()
     {
         $this
             ->setDefinition(array(
-                new InputArgument('filter', InputArgument::OPTIONAL, 'Show details for all entries matching this filter'),
+                new InputArgument('name', InputArgument::OPTIONAL, 'The template name'),
+                new InputOption('filter', null, InputOption::VALUE_REQUIRED, 'Show details for all entries matching this filter'),
                 new InputOption('format', null, InputOption::VALUE_REQUIRED, 'The output format (text or json)', 'text'),
             ))
             ->setDescription('Shows a list of twig functions, filters, globals and tests')
             ->setHelp(<<<'EOF'
 The <info>%command.name%</info> command outputs a list of twig functions,
-filters, globals and tests. Output can be filtered with an optional argument.
+filters, globals and tests.
 
   <info>php %command.full_name%</info>
 
 The command lists all functions, filters, etc.
 
-  <info>php %command.full_name% date</info>
+  <info>php %command.full_name% @Twig/Exception/error.html.twig</info>
+
+The command lists all paths that match the given template name.
+
+  <info>php %command.full_name% --filter=date</info>
```

Comment:
This changes the command completely. Not sure if we want to keep BC on commands (I think we do not), but that means we need a PR on docs as well.

Reply:
I'll do it as soon as possible.